compliance_suite/job_runner.py

```diff
@@ -167,8 +170,8 @@ def run_jobs(self) -> None:
         for yaml_file in os.listdir(yaml_path):
             if yaml_file.endswith(".yml"):
                 self.test_count += 1
-                logger.log(LOGGING_LEVEL['SUMMARY'], "\n{:#^100}".format(f"     Initiating Test-{self.test_count}"
-                                                                         f" for {yaml_file}     "))
+                logger.summary("\n" + PATTERN_HASH_CENTERED.format(
+                    f"     Initiating Test-{self.test_count} for {yaml_file}     "))
```

Comment:
Logging in a loop with both `+` and f-strings is probably going against all the recommendations that pylint has in terms of logging :laughing:

Couldn't you construct your string first (and do any necessary concatenations with `.join()`) and then use a single, simple `logger.summary()` call?